doc/snippets/Microsoft.Data.SqlClient/SqlConnection.xml

```diff
@@ -129,6 +129,21 @@ using (SqlConnection connection = new SqlConnection(connectionString))
             <value>The access token for the connection.</value>
             <remarks>To be added.</remarks>
         </AccessToken>
+        <AccessTokenCallback>
```

Comment:
Can you add the expected exception as well?

Reply:
Since this is a callback provided by the developer user, it could be anything 😄

Reply:
That's right. I don't mean the application's exceptions, you can include the exceptions that you expect to happen and already have handled.

Reply:
I must be missing something - the only exceptions that can occur would be the ones generated by the contents of the callback, which are supplied by the user. Which exceptions did you have in mind that we would expect or already have handled?

Reply:
`InvalidOperationException` and `ArgumentNullException` from the AccessTokenCallback's setter.